src/main/java/colorize/views/GUI.java

```diff
@@ -120,11 +156,103 @@ public GUI(Controller pContr)
 
 		frame.setVisible(true);
 	}
+	
+	public boolean getBacktracking()

```

Comment:
Zu umständlich für den Controller, besser eine Methode pro Radio-Gruppe die den Zustand (z.B. als Enum) zurückgibt
```suggestion
	public Algorithm getAlgorithm()
```